nixos/lib/test-driver/test_driver/machine.py

```diff
@@ -946,6 +943,15 @@ def crash(self) -> None:
         self.send_monitor_command("quit")
         self.wait_for_shutdown()
 
+    def reboot(self) -> None:
+        """Press Ctrl+Alt+Delete in the guest.
+
+        Prepares the machine to be reconnected which is useful if the
+        machine was started with `allow_reboot = True`
```

Comment:
would it make sense to throw an exception if `allow_reboot` was not set to `True`? Otherwise, what state would the object be in if used like this?

Reply:
qemu would exit which is a valid use-case for a test case.

Reply:
Ok, i just played around with it. We get a broken pipe exception in the `start()` method. That should be good enough for the beginning.